flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlAlterTableOptions.java

```diff
@@ -29,22 +29,33 @@
 
 import static java.util.Objects.requireNonNull;
 
-/** ALTER TABLE [[catalogName.] dataBasesName].tableName SET ( name=value [, name=value]*). */
+/**
+ * ALTER TABLE IF EXISTS [[catalogName.] dataBasesName].tableName SET ( name=value [, name=value]*).
+ */
 public class SqlAlterTableOptions extends SqlAlterTable {
 
     private final SqlNodeList propertyList;
 
     public SqlAlterTableOptions(
-            SqlParserPos pos, SqlIdentifier tableName, SqlNodeList propertyList) {
-        this(pos, tableName, null, propertyList);
+            SqlParserPos pos, boolean ifExists, SqlIdentifier tableName, SqlNodeList propertyList) {
```

Comment:
Maybe change to `SqlAlterTableOptions(SqlParserPos, SqlIdentifier, SqlNodeList, boolean)`?